lms/djangoapps/teams/static/teams/templates/team-actions.underscore

```diff
@@ -0,0 +1,4 @@
+<div class="team-actions">
+  <h3 class="title"><%= gettext("Are you having trouble finding a team to join?") %></h3>
```

Comment:
Please use `<%-` when injecting the output of `gettext`, since it does HTML escaping, unlike `<%=`.